src/lang-promql/promql.ts

```diff
@@ -0,0 +1,28 @@
+import { LezerSyntax } from "@codemirror/next/syntax";
+import { parser } from "lezer-promql";
+import { styleTags } from "@codemirror/next/highlight";
+import { Extension } from "@codemirror/next/state";
+
+export const promQLSyntax = new LezerSyntax(parser.withProps(
+  styleTags({
+    LineComment: "comment",
+    LabelName: "name",
```

Comment:
This can be more specific, `labelName`.

Reply:
well I can add it np, but it won't change anything normally. Since labelName is linked to the name

https://github.com/codemirror/codemirror.next/blob/master/highlight/src/highlight.ts#L166